network/src/network.rs

```diff
@@ -192,14 +191,13 @@ impl NetworkState {
                 duration.as_secs(),
                 reason
             );
-            metrics!(
-                counter,
-                "ckb.network.ban_peer",
-                1,
-                "peer_addr" => addr.to_string(),
-                "duration" => duration.as_secs().to_string(),
-                "reason" => reason.clone(),
-            );
+            ckb_metrics::CKB_NETWORK_BAN_PEER
+                .with_label_values(&[
+                    addr.to_string().as_str(),
+                    duration.as_secs().to_string().as_str(),
+                    &reason,
+                ])
+                .inc();
```

Comment:
Do we really need to expose `addr`,`duration`, and `reason` to prometheus? 
I think it's a potential point of high memory consumption, it will cause OOM in the worst case.

Reply:
No, we can search that in log.